### DISCUSSION TITLE
remove the 'x' and 'y' object parameters and convert to Array to save space

### DISCUSSION THREAD

**Discussion by Alphazork** [razztyfication/vue-drawing-canvas#19] [Ideas]
Currently when saving all Strokes for later use the object is very big. 
Would it be possible to convert the coordinates array of objects to a multi dimensional array?
current:
`{"x": 330,"y": 154},`
will become:
`[330,154]`

**Comment by razztyfication** [comment]
you can do that by yourself.

using [lodash](https://lodash.com/docs#reduce) maybe:
```js
_.reduce([{"x": 330, "y": 154}], (coordinateArray, coordinate) => {
    coordinateArray.push([coordinate.x, coordinate.y])
    return coordinateArray
}, [])
```

to use it for props:
```js
_.reduce([[330,154]], (coordinateArray, coordinate) => {
    coordinateArray.push({"x": coordinate[0], "y": coordinate[1]})
    return coordinateArray
}, [])
```

**Reply by Alphazork** [reply]
Thank you for the suggestion. Yes that would be possible but would be one more step. Why not include it directly and save time and processing power.
In my testing the size for one drawing went from 140kB to 98 kB! this is huge!
